### PR TITLE
Example application code updates to support Swarm Callback additional parameters

### PR DESCRIPTION
**Pull request by iArpanPatel**
In SwarmCallBack following parameters are provided to enable displaying training progress or ETA of training on the SLM UI.

- 'lossFunction' - 'lossFunction' string should match with loss function class defined in PyTorch -
   https://pytorch.org/docs/stable/nn.html#loss-functions

- 'lossFunctionArgs'  - dictionary of named arguments for lossFunction as shown below.

- 'metricFunction'    - 'metricFunction' string should match with metric function class defined in torchmetrics -
   https://torchmetrics.readthedocs.io/en/stable/all-metrics.html

- 'metricFunctionArgs'- dictionary of named arguments for metricFunction as show below.

- 'totalEpochs'       - Total epochs used in local training.

### CODE REVIEW

**Comment by iArpanPatel** [comment]
Copyright year in cifar10.py updated.

**Review by girisairam450 — APPROVED** [review]
Looks good.
